acceptance-tests/src/test/java/uk/gov/di/test/step_definitions/LockoutsStepDef.java

```diff
@@ -142,6 +143,14 @@ public void theLockoutTextIsDisplayed(String lockoutText) {
 
     @And("the {string} retry Text is displayed")
     public void theRetryTextIsDisplayed(String reTryTxt) {
-        assertEquals(reTryTxt, getRetryText());
+        String actualFullText = (getRetryText());
+        String delimiter = "\n";
+        String actualText = Arrays.toString(actualFullText.split(delimiter));
```

Comment:
There's little benefit having a variable for such a simple delimiter.  Consider just doing this:

String actualText = Arrays.toString(actualFullText.split("\n"));